ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2135,5 +2135,81 @@
         "name": "type-java.svg",
         "mediaType": "image/svg+xml"
     }
+  },
+  {
+    "id": "spring-boot",
+    "creator": "Gytis Trikleris",
+    "name": "Spring Boot",
+    "description": "Spring Boot Stack on CentOS.",
+    "scope": "advanced",
+    "tags": [
+      "Java",
+      "JDK",
+      "Maven",
+      "Spring Boot",
+      "CentOS",
+      "Git"
+    ],
+    "components": [
+      {
+        "name": "CentOS",
+        "version": "---"
+      },
+      {
+        "name": "JDK",
+        "version": "1.8.0_45"
+      },
+      {
+        "name": "Maven",
+        "version": "3.2.2"
+      }
+    ],
+    "source": {
+      "type": "image",
+      "origin": "registry.centos.org/che-stacks/spring-boot"
+    },
+    "workspaceConfig": {
+      "name": "default",
+      "defaultEnv": "default",
+      "environments": {
+        "default": {
+          "recipe": {
+            "location": "registry.centos.org/che-stacks/spring-boot",
+            "type": "dockerimage"
+          },
+          "machines": {
+            "dev-machine": {
+              "agents": [
+                "org.eclipse.che.terminal",
+                "org.eclipse.che.ws-agent",
+                "org.eclipse.che.ssh"
+              ],
+              "attributes": {
+                "memoryLimitBytes": "2147483648"
+              }
+            }
+          }
+        }
+      },
+      "commands": [
+        {
+          "commandLine": "scl enable rh-maven33 'mvn clean install -f ${current.project.path}'",
+          "name": "build",
+          "type": "custom"
+        },
+        {
+          "commandLine": "java -jar ${current.project.path}/target/*.jar",
+          "name": "run",
+          "type": "custom",
+          "attributes": {
+            "previewUrl": "http://${server.port.8080}"
+          }
+        }
+      ]
+    },
+    "stackIcon": {
+      "name": "type-java.svg",
```

Comment:
@gytis I'm happy to help you on that - no problem.
Could you tell me which icon do you wan to use?
- The spring leaf: https://en.wikipedia.org/wiki/File:Pivotal_Java_Spring_Logo.png
- The spring "power" button: http://stormpath.com/wp-content/uploads/2016/05/spring-boot-logo.jpg

Also, if you can point me to an SVG file, that would be ideal.